src/screens/DiscoveryScreenV2/elements/DiscoveryTab.js

```diff
@@ -1,44 +1,62 @@
 import * as React from 'react';
 import {Dimensions, Keyboard, Pressable, ScrollView, StyleSheet, Text, View} from 'react-native';
 
+import {useRoute} from '@react-navigation/core';
 import {colors} from '../../../utils/colors';
 import {fonts, normalizeFontSize} from '../../../utils/fonts';
+import {setNavbarTitle} from '../../../context/actions/setMyProfileAction';
+import {Context} from '../../../context';
 
-const {width} = Dimensions.get('screen');
+const windowWidth = Dimensions.get('window').width;
+
+const DiscoveryTab = ({onChangeScreen, selectedScreen = 0, tabs}) => {
+  const [, dispatchNavbar] = React.useContext(Context).profile;
+  const route = useRoute();
 
-const DiscoveryTab = ({onChangeScreen, selectedScreen = 0}) => {
   const handleTabOnClicked = React.useCallback((index) => {
     Keyboard.dismiss();
     onChangeScreen(index);
   }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleTabOnClicked` function has a dependency on `onChangeScreen` but it's not included in the dependency array of `useCallback`. This could lead to unexpected behavior if `onChangeScreen` changes.
```diff
-    const handleTabOnClicked = React.useCallback((index) => {
-      Keyboard.dismiss();
-      onChangeScreen(index);
-    }, []);

+    const handleTabOnClicked = React.useCallback((index) => {
+      Keyboard.dismiss();
+      onChangeScreen(index);
+    }, [onChangeScreen]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->